content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md

```diff
@@ -10,4 +10,69 @@ compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
 familiar with the use and operation of Istio 1.4.
 
-TBD
+## Control Plane Restructuring
```

Comment:
Can we include a mapping between the components and `istiod`, Envoy, etc?

Reply:
If there are no glaring issues can we merge this and iterate improvements? if we keep making these megaPRs with 100s of comments it's very hard to follow